src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexMatchTimeoutException.cs

```diff
@@ -32,15 +32,15 @@ public RegexMatchTimeoutException(string regexInput, string regexPattern, TimeSp
         /// developers should prefer using the constructor
         /// <code>public RegexMatchTimeoutException(string input, string pattern, TimeSpan matchTimeout)</code>.
         /// </summary>
-        public RegexMatchTimeoutException() { }
+        public RegexMatchTimeoutException() : base(SR.RegexMatchTimeoutException_Occurred) { }
 
         /// <summary>
         /// This constructor is provided in compliance with common .NET Framework design patterns;
         /// developers should prefer using the constructor
         /// <code>public RegexMatchTimeoutException(string input, string pattern, TimeSpan matchTimeout)</code>.
         /// </summary>
         /// <param name="message">The error message that explains the reason for the exception.</param>
-        public RegexMatchTimeoutException(string message) : base(message) { }
+        public RegexMatchTimeoutException(string? message) : base(message ?? SR.RegexMatchTimeoutException_Occurred) { }
```

Comment:
Do not change the public API.
```suggestion
        public RegexMatchTimeoutException(string message) : base(message ?? SR.RegexMatchTimeoutException_Occurred) { }
```